Protocol.md

```diff
@@ -273,6 +273,8 @@ rules MAY be used for handling layout. For example, `application/x.netflix+cbor`
 * __Setup Data__: includes payload describing connection capabilities of the endpoint sending the
 Setup header.
 
+__NOTE__: A server that receives a SETUP frame that has (__R__)esume Enabled set, but does not support resuming operation, must reject the SETUP with an ERROR. 
```

Comment:
I question the value of not having a SETUP ACK, mainly because we have specified typical keepalives are subsecond.  A client that wants to connect to a server and just send metadata push and fireAndForget could be talking to a brick wall for all it knows.